### ISSUE TITLE
Status of things and questions about version 3

### ISSUE THREAD

**Issue by FSharpCSharp** [dotnet/docfx#5298]
I have to ask a question about the current development status of version 3, because the roadmap was only updated a quarter ago, and you haven't heard much since.

What is the current status of version 3? Is there already a rough timetable when you can expect a version that can at least create help pages? So far, only the metadata was processed, but the actual content of the Markdown documents was not.

It would be good to know by when an approximate alpha version would be released, which could also be used to visually display content. That would be a milestone.

Basically, you get the impression that the development is progressing very slowly. That's actually a pity, because docfx is incredibly practical.

**Comment by cillroy** [comment]
Due to the complexity of revamping a large amount of the DocFX internals, we are looking to produce an new artifact in early 2020

**Comment by FSharpCSharp** [comment]
Thank you very much for the quick answer! That answers my question, and so we can close this issue. Many thanks again!
